patches/tModLoader/Terraria/Player.cs.patch

```diff
@@ -315,6 +315,25 @@
  					for (int j = 0; j < 22; j++)
  					{
  						if (num == 27 && (this.buffType[j] == num || this.buffType[j] == 101 || this.buffType[j] == 102))
+@@ -2173,6 +_,10 @@
+ 					}
+ 					if (this.inventory[i].mana > 0 && flag)
+ 					{
++						if (this.CheckMana(this.inventory[i], this.GetManaCost(this.inventory[i]), true, true)) {
++							this.manaRegenDelay = (int)this.maxRegenDelay;
++						}
++						goto SkipManaCode;
```

Comment:
nice goto, I should consider using that to keep patches down

Reply:
Thanks, and yeah, to me gotos seem to be the way to go when wanting to reduce patch size